### ISSUE TITLE
License

### ISSUE THREAD

**Issue by SteveClement** [LateRegistration/lr-restify#35]
Dear GoC Team,

Please add a LICENSE file so everyone knows how to share, implement, re-use, distribute, modify, improve your code.

It is not the easiest of choices but here is a good start: http://choosealicense.com/

Once chosen just put a file called LICENSE in the root of your Repository :)

Thanks a lot for having been part of the first Game of Code Hackathon, you were AWESOME!

Steve

**Comment by websoftwares** [comment]
Dear @SteveClement

We will add the MIT license and since we have created several rest API methods for the application we demonstrated we will publish them or free today.

This includes:
- find veloh bikes closest to my location (lat,long) with real time information.
- find busses closest to my location (lat,long) with real time information.
- find your city cityhall information.

Also we encourage people to re-use and host themselves the dockerized rest server because we will promote for social sharing of architecture for horizontal and vertical scaling.

Sincerely,
Boris Verhaaff
